src/main/kotlin/no/nav/familie/ef/sak/behandling/fremleggsoppgave/FremleggsoppgaveController.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.behandling.fremleggsoppgave
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/fremleggsoppgave")
+@ProtectedWithClaims(issuer = "azuread")
+class FremleggsoppgaveController(private val fremleggsOppgaveService: FremleggsoppgaveService) {
+
+    @GetMapping("/kanopprettes/{behandlingid}")
```

Comment:
Her tenker jeg man henter informasjon "fremleggsoppgave" for behandling X, og då spesifikt kan-opprettes-informasjon
```suggestion
    @GetMapping("{behandlingid}/kan-opprettes")
```

(Vet ennå ikke hva kan-opprettes egentlige returnerer)